### PR TITLE
Modify get donors and who liked projects query to exclude somethings

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
https://github.com/Giveth/notification-center/issues/17

### CODE REVIEW

File: src/repositories/donationRepository.ts [hunk 1]
```diff
@@ -74,16 +76,23 @@ export const findDonationById = async (
...
+      'reaction',
+      'reaction.projectId = project.id AND user.id = reaction.userId',
+    )
     .distinctOn(['user.walletAddress'])
     .select('LOWER(user.walletAddress) AS "walletAddress", user.email as email')
-    .where(`"projectId"=:projectId`, {
+    .where(`donation."projectId"=:projectId`, {
       projectId,
     })
+    .andWhere(`reaction.id IS NULL`)
```
Comment (CarlosQ96) [inline src/repositories/donationRepository.ts:95]: Hmm not sure about this condition or the test it self. Will work on double checking this.

**Review by CarlosQ96 — COMMENTED** [review]
Excluding the owner looks good too me, ill work on the exclude the person that liked in that moment. I think the test and condition is not clear... or maybe is the naming or I am confusing myself haha

File: src/repositories/donationRepository.ts (hunk 1 above)
Comment (CarlosQ96) [reply src/repositories/donationRepository.ts:95]: Should it be instead?
```
    .andWhere(`reaction.userId != donation.userId`) 
```

Comment (CarlosQ96) [reply src/repositories/donationRepository.ts:95]: Or do you mean users who did not like the project?

**Comment by mohammadranjbarz** [comment]
@CarlosQ96 
Thanks for your review, we need to deploy it now, so I merge it, but we will fix it in the future

**Pull request merged**